packages/core/src/components/menu/_menu.scss

```diff
@@ -61,6 +61,7 @@ Styleguide pt-menu
   &::before,
   .pt-icon {
     align-self: flex-start;
```

Comment:
that flex-start works for the icon on the left side, but not for the item label on the right. can you also add flex start for `.pt-menu-item-label` on line 69?

Reply:
Done.